FreeRTOS_IP.c

```diff
@@ -2099,7 +2105,8 @@ uint16_t usGenerateChecksum( uint32_t ulSum, const uint8_t * pucNextData, size_t
 xUnion32 xSum2, xSum, xTerm;
 xUnionPtr xSource;		/* Points to first byte */
 xUnionPtr xLastSource;	/* Points to last byte plus one */
-uint32_t ulAlignBits, ulCarry = 0ul;
+uint64_t ulAlignBits;
```

Comment:
Should be safe, its a max 2 bit wide result thing that it tracks. Looks good to me.